.github/workflows/release-pages.yml

```
@@ -21,10 +18,11 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+      - name: Add Flutter to PATH
+        run: echo "/home/ubuntu/flutter/bin" >> $GITHUB_PATH
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:36: Double quote to prevent globbing and word splitting

(shellcheck)